plugins/lookup/passwordstore.py

```diff
@@ -106,6 +106,22 @@
         type: str
         default: 15m
         version_added: 4.5.0
+      backend:
+        description:
+          - Specify which backend to use.
+          - Defaults to C(pass), passwordstore.org's original pass utility.
+          - C(gopass) support is incomplete.
+        ini:
+          - section: passwordstore_lookup
+            key: backend
+        vars:
+          - name: passwordstore_backend
+        type: str
+        default: pass
+        choices:
+          - pass
+          - gopass
+        version_added: 4.5.0
```

Comment:
The next release is 5.2.0:
```suggestion
        version_added: 5.2.0
```